tests/parsers/test_pw.py

```diff
@@ -595,6 +595,32 @@ def test_pw_vcrelax_success_fractional(
     })
 
 
+def test_pw_vcrelax_success_rVV10(
+    aiida_profile, fixture_localhost, generate_calc_job_node, generate_parser, generate_inputs, data_regression
+):
+    """Test a `vc-relax` rVV10 run that successfully converges."""
+    name = 'vcrelax_success_rVV10'
+    entry_point_calc_job = 'quantumespresso.pw'
+    entry_point_parser = 'quantumespresso.pw'
+
+    inputs = generate_inputs(calculation_type='vc-relax')
+    node = generate_calc_job_node(entry_point_calc_job, fixture_localhost, name, inputs)
+    parser = generate_parser(entry_point_parser)
+    results, calcfunction = parser.parse_from_node(node, store_provenance=False)
+
+    assert calcfunction.is_finished, calcfunction.exception
+    assert calcfunction.is_finished_ok, calcfunction.exit_message
+    assert not orm.Log.objects.get_logs_for(node), [log.message for log in orm.Log.objects.get_logs_for(node)]
+    assert 'output_parameters' in results
+    assert 'output_structure' in results
+    assert 'output_trajectory' in results
+    data_regression.check({
+        'output_parameters': results['output_parameters'].get_dict(),
+        'output_structure': results['output_structure'].attributes,
+        'output_trajectory': results['output_trajectory'].attributes,
+    })
```

Comment:
Since we are checking a very specific part of the parsing code here, I would argue that it is not necessary to check the entire parsed output, but just the part that is relevant here. The parsing of the rest of the information is already done in the default tests. So what we could check here is:
* the `energy_vdw` key is in `output_parameters`
* the `output_trajectory` contains the parsed `stress`.

That would be sufficient and saves a lot of unnecessary lines in the comparison files. Other than that everything looks good.

Reply:
Just to check: Since the stress array cannot be represented in `.yml` format, it's ok just check the array size stored in the attributes of the `output_trajectory`, right?